portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+/** Models a task that the user creates. */
+public final class Task {
+
+  private final String name;
+  private final String description;
+  private final Integer durationMinute;
```

Comment:
Will all tasks have the associated duration? If not, consider making this field optional.

Reply:
All tasks will have a duration so we have a time to fit within the calendar.